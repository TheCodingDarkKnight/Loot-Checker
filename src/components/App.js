import React, { Component } from "react";

import Wallet from "./Wallet";
import Loot from "./Loot";

const App = () => {
    return (
        <div>
            <h2>Loot check</h2>
            <hr />
            <Wallet />
            <Loot />
            <div>Powered by <a href="https://www.coindesk.com/price" target="_blank" without rel="noopener noreferrer">Coindesk</a></div>
        </div>
    )
}

export default App;