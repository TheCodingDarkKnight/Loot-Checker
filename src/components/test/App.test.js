import React from "react";
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

import App from "../App";

configure({ adapter: new Adapter() });

describe("App", () => {
    const app = shallow(<App />);

    it("Renders properly", () => {
        expect(app).toMatchSnapshot();
    });

    it("Contains a connected Wallet component", () => {
        console.log(app.debug())
        expect(app.find("Connect(Wallet)").exists()).toBe(true);
    })
})