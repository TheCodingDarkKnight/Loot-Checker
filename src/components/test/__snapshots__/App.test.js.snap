// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Renders properly 1`] = `ShallowWrapper {}`;
